apps/hub/src/app/_constants/address.ts

```diff
@@ -29,3 +32,18 @@ export const FAUCET = {
   address: '0x4Fb609F4a457f47B41D35Dd060447271F000120A' as Address,
   abi: faucetAbi as Abi,
 } as const
+
+export const KARMA = {
+  address: '0x7ec5Dc75D09fAbcD55e76077AFa5d4b77D112fde' as Address,
+  abi: karmaAbi as Abi,
+} as const
+
+export const REWARDS = {
+  address: '0xAEF19bbbe490Ad9C083EcE40c835A2f21B720de8' as Address,
+  abi: rewardsAbi as Abi,
+} as const
+
+export const KARMA_TIER = {
+  address: '0x011b9de308BE357BbF24EfB387a270a14A04E5d2' as Address,
```

Comment:
wrong address, the implementation is at `0xc7fCD786a161f42bDaF66E18a67C767C23cFd30C` now

Reply:
https://github.com/status-im/status-web/pull/837/commits/944b41d0528a3d61feb662a0132abe77e331819d